src/libraries/System.Private.CoreLib/src/System/Globalization/CultureData.cs

```diff
@@ -2094,7 +2094,7 @@ private static string GetSeparator(string format, string timeParts)
         private static int IndexOfTimePart(string format, int startIndex, string timeParts)
         {
             Debug.Assert(startIndex >= 0, "startIndex cannot be negative");
-            Debug.Assert(timeParts.IndexOfAny(new char[] { '\'', '\\' }) == -1, "timeParts cannot include quote characters");
+            Debug.Assert(timeParts.IndexOfAny(new char[] { '\'', '\\' }) < 0, "timeParts cannot include quote characters");
```

Comment:
I know that this array is not allocated in `Release` builds, but it still hurts my eyes

```diff
- timeParts.IndexOfAny(new char[] { '\'', '\\' })
+ timeParts.AsSpan().IndexOfAny('\'', '\\')`
```

Reply:
https://github.com/dotnet/runtime/pull/63852/files#diff-9258525d3e30b5aa688681a2bad2283d01b556cc745380f0f4e9d6ac2550231dR2097